test_suite/package/test_efs_utils.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from lib import test_lib
+
+
+@pytest.mark.package
+@pytest.mark.run_on(['rhel9.4'])
+class TestsEfsUtils:
+    def test_efs_utils(self, host, instance_data):
+        """
+        NOTE: This test case assumes efs-utils is preinstalled in the AMI image
+        Check basic functionality of EFS utils:
+        - Create a mount point and mount the EFS file system to the RHEL instance.
+        - Create a file on the mount point.
+        - Automate mount on boot.
+        - Checksum the file before reboot.
+        - reboot and checksum that the file is correct.
+        """
+        # will get 'file_system_dns_name' arg form the fixture that gets the output of the Terraform script
+        file_system_dns_name = instance_data['efs_file_system_dns_name']
+        test_file = '/mnt/efs/testfile'
```

Comment:
I would be cautious with the file name. It can have issue when running on more than 1 instance belonging to the same region, they will share the efs file system and if they both try to write on the same file, it can have unexpected results.

I propose to append a random string at the end of the filename, or to the whole filename.

E.g.:
```
import uuid

...

test_file = '/mnt/efs/{}'.format(str(uuid.uuid4()))
```

Reply:
Added EPOCH timestamp instead.